dangerzone/container_utils.py

```diff
@@ -0,0 +1,149 @@
+import gzip
```

Comment:
I'm wondering about the name, if we should turn `utils` into a module, and have `utils.container` maybe? Not a big deal, I'll let you decide.

Reply:
Yeah, I don't like `container_utils.py` as well. I guess I'd sit on it for now, but once we start having more utils like this, we should probably namespace them.